app/command.go

```diff
@@ -191,6 +191,100 @@ func (a *App) ExecuteCommand(args *model.CommandArgs) (*model.CommandResponse, *
 	return nil, model.NewAppError("command", "api.command.execute_command.not_found.app_error", map[string]interface{}{"Trigger": trigger}, "", http.StatusNotFound)
 }
 
+type mentionMapItem struct {
+	Name string
+	Id   string
+}
+
+// mentionsToTeamMembers returns all the @ mentions found in message that
```

Comment:
After reading through the existing mention parsing code, I fully understand the need to roll something custom here. That being said -- and separate from this PR -- it would be nice to refactor the code to support the "one true way".

Reply:
I completely agree. The code regarding mentions is a bit scattered (I ended up looking into mattermost-webapp, actualli)y so I am all in for a single source of truth.